style-guide.md

```diff
@@ -205,12 +205,12 @@ abbreviation as a part of a longer name. For example, write `HttpServer` instead
 of `HTTPServer`.
 
 Unicode symbols are allowed only in modules that already use unicode symbols. If
-you create unicode name, you should also create a non-unicode one as an alias.
+you create a unicode name, you should also create a non-unicode one as an alias.
 
 ### Data types
 
 Creating data types is extremely easy in Haskell. It is usually a good idea to
-introduce custom data type (enum or `newtype`) instead of using commonly used
+introduce a custom data type (enum or `newtype`) instead of using a primitive
```

Comment:
This rule is not only about primitive types. It should be applied even to more complex types like `State`, `Reader` or just `Set Int`.

Reply:
Oh okay, my bad. I looked at the example and made an assumption 😅